app/MMB/src/scenes/timeline/events/BoardingTimelineEvent.js

```diff
@@ -0,0 +1,46 @@
+// @flow
+
+import React from 'react';
+import idx from 'idx';
+import { graphql, createFragmentContainer } from '@kiwicom/mobile-relay';
+import { Translation } from '@kiwicom/mobile-localization';
+import { Text, Color, StyleSheet } from '@kiwicom/mobile-shared';
+
+import type { BoardingTimelineEvent as BoardingTimelineEventType } from './__generated__/BoardingTimelineEvent.graphql';
+import TimelineEvent from '../TimelineEvent';
+import TimelineEventIcon from '../TimelineEventIcon';
+
+type Props = {|
+  +data: ?BoardingTimelineEventType,
+|};
+
+const BoardingTimelineEvent = (props: Props) => {
+  const timestamp = idx(props, _ => _.data.timestamp);
+  return (
+    <TimelineEvent
+      timestamp={timestamp}
+      iconVertLines={<TimelineEventIcon type="TextIcon" value="&#xe00a;" />}
+      mainContent={
+        <React.Fragment>
+          <Translation id="mmb.booking_timeline.event.gate_arrival.title" />
+          <Text style={styles.note}>
+            <Translation id="mmb.booking_timeline.event.gate_arrival.note" />
+          </Text>
+        </React.Fragment>
+      }
+    />
+  );
+};
+
+export default createFragmentContainer(
+  BoardingTimelineEvent,
+  graphql`
+    fragment BoardingTimelineEvent on BoardingTimelineEvent {
+      timestamp
+    }
+  `,
+);
+
+const styles = StyleSheet.create({
+  note: { fontSize: 13, color: Color.ink.light },
```

Comment:
I should probably be an eslint-rule, that is much more readable

```
note: { 
fontSize: 13, 
color: Color.ink.light,
},
```

Reply:
We can add 
```
"object-property-newline": ERROR
```
to `.eslintrc.js` to fix this. I agree with you.

Reply:
Yes, please do so

Reply:
I'll make a different PR for this, it is modifying too many files after I add this rule.

Reply:
Right, ok, then I would say leave it for now. Not that important. 

Some times it makes sense to write it on one line I guess.

Reply:
The notes are taken care of in the other PR https://github.com/kiwicom/mobile/pull/762